shared/api/network_peer.go

```diff
@@ -61,6 +96,13 @@ type NetworkPeer struct {
 	// Example: network1
 	TargetNetwork string `json:"target_network" yaml:"target_network"`
 
+	// lxdmeta:generate(entities=network-peering; group=peering-properties; key=status)
+	// This property is read-only and cannot be updated.
```

Comment:
We still want the information you had in there before ("Indicates if mutual peering exists with the target network."), plus the read-only information.
Just not the part that was copy&paste and unrelated. ;)

Reply:
fixed